previousnext/m8s
# Improve contributor documentation.

#### What does this PR do?

* Adds some improved contributor documentation
* Adds a ready-to-go development environment 

#### How should this be manually tested?

1. Check out this branch
1. Run a few commands to test.

```bash
docker-compose run --rm dev make lint
docker-compose run --rm dev make test
docker-compose run --rm dev make build
```

#### What are the relevant tickets?

https://redmine.previousnext.com.au/issues/47648
